pkg/manifest/subscription.go

```diff
@@ -123,7 +123,7 @@ func subscriptionService(subscriptionOptions subscription.ImageOptions, serviceO
 		if subscriptionOptions.TemplateUUID != "" {
 			curlToAssociateSystem = getCurlToAssociateSystem(subscriptionOptions)
 		} else if subscriptionOptions.TemplateName != "" {
-			rhcConnect += fmt.Sprintf(" --content-template %s", subscriptionOptions.TemplateName)
+			rhcConnect += fmt.Sprintf(" --content-template=\"%s\"", subscriptionOptions.TemplateName)
```

Comment:
Nitpick: you could have used `%q` instead of `\"%s\"`.

Reply:
ohp! TIL :)  i'll keep that in mind for next time, thank you :heart: